packages/fluentui/docs/src/examples/components/Toolbar/Content/ToolbarExampleCustomContent.shorthand.tsx

```diff
@@ -1,7 +1,51 @@
 import * as React from 'react';
-import { Button, Text, Toolbar } from '@fluentui/react-northstar';
+import { Button, Text, Toolbar, SplitButton } from '@fluentui/react-northstar';
 import { BoldIcon } from '@fluentui/react-icons-northstar';
 
+// used to prevent arrow navigation while SplitButton menu is open.
+function onItemKeyDown(e) {
+  if (e.key === 'ArrowLeft' || e.key === 'ArrowRight') {
```

Comment:
Should this be part of `splitButtonBehavior`?

Reply:
I don't think so, it's just a custom requirement. At most it can be part of a separate behavior.

Reply:
I'd vote to create a separate behavior for that, @jurokapsiar wdyt?